### PR TITLE
Fix issues in windows

### PR DESCRIPTION
**Pull request by ssrlive**

### CODE REVIEW

**Review by sseemayer — APPROVED** [review]

File: src/key.rs [hunk 1]
```diff
@@ -69,7 +69,11 @@ fn parse_xml_keyfile(xml: &[u8]) -> Result<KeyElement, DatabaseKeyError> {
     if key_version == Some("2.0".to_string()) {
         // TODO we should also validate the integrity of a v2 keyfile using the hash value
 
-        let trimmed_key = key_value.trim().replace(" ", "").replace("\n", "");
+        let trimmed_key = key_value
+            .trim()
+            .replace(" ", "")
+            .replace("\n", "")
+            .replace("\r", "");
```
Comment (louib) [inline src/key.rs:76]: @ssrlive We should probably add a unit test for this.

Comment (ssrlive) [reply src/key.rs:76]: image

Comment (louib) [reply src/key.rs:76]: But this is a test run from your branch and your fork. We don't have a unit test for this upstream, right?

Comment (louib) [reply src/key.rs:76]: @ssrlive gotcha, I thought we needed a new keyfile to test this, but looks like we can reproduce by running the tests on Windows.

**Pull request merged**